pkg/testing/integration/program.go

```diff
@@ -560,9 +560,6 @@ func (opts ProgramTestOptions) With(overrides ProgramTestOptions) ProgramTestOpt
 	if overrides.PrepareProject != nil {
 		opts.PrepareProject = overrides.PrepareProject
 	}
-	if overrides.LocalDependencies != nil {
```

Comment:
Any reason for removing this from the "overrides" options?

Reply:
Oh nope that was a mistake will readd it, good catch